models/users.js

```diff
@@ -0,0 +1,166 @@
+'use strict';
+
+var Db        = require('/../config/database');
+var crypto    = require('crypto');
+var validator = require('validator');
+
+var UserSchema = mongoose.Schema({
+    username : { type: String, required: true },
+    email    : { type: String, required: true },
+    password : { type: String, required: true },
+    token    : { type: String, required: true }
```

Comment:
Tokens aren't stored with the user. We're storing tokens in a separate database.